templates/field/field--paragraph--field-service-links--popular-service-item.html.twig

```diff
@@ -0,0 +1,5 @@
+<ul class="popular-service-item__links">
```

Comment:
Previously code like this has bitten our behind when for some reason this `ul` has had no items. Please add an `if` -check to verify that there actually are more than 0 items incoming.

Reply:
Good point! Added this check!